packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -85,61 +123,71 @@ foreman-installer --scenario katello --migrations-only > /dev/null
 # RPM can't change a directory into a symlink
 # https://bugzilla.redhat.com/show_bug.cgi?id=447156
 for scenario in foreman-proxy-content katello ; do
-	MIGRATIONS=%{_sysconfdir}/%{name}/scenarios.d/$scenario.migrations
+	MIGRATIONS=%{scenariodir}/$scenario.migrations
 	if [ -d $MIGRATIONS ] && [ ! -L $MIGRATIONS ] ; then
-		mv $MIGRATIONS/.applied %{_sysconfdir}/%{name}/scenarios.d/$scenario-migrations-applied
+		mv $MIGRATIONS/.applied %{scenariodir}/$scenario-migrations-applied
 		rm -rf $MIGRATIONS
 	fi
 done
 
-%files
+%files common
 %defattr(-,root,root,-)
 %doc README.*
 %license LICENSE
 %dir %{_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/%{name}/custom-hiera.yaml
-%dir %{_sysconfdir}/%{name}/scenarios.d
-%{_sysconfdir}/%{name}/scenarios.d/foreman.migrations
-%config(noreplace) %attr(600, root, root) %{_sysconfdir}/%{name}/scenarios.d/foreman.yaml
-%config(noreplace) %attr(600, root, root) %{_sysconfdir}/%{name}/scenarios.d/foreman-answers.yaml
-%config(noreplace) %{_sysconfdir}/%{name}/scenarios.d/foreman-migrations-applied
+%dir %{scenariodir}
 %{_sbindir}/%{name}
-%{_datadir}/%{name}
-%{_mandir}/man8
-
-# katello files
-%exclude %{_datadir}/%{name}/config/foreman-proxy-content*
-%exclude %{_datadir}/%{name}/config/katello*
-%exclude %{_datadir}/%{name}/katello-certs
-%exclude %{_datadir}/%{name}/parser_cache/foreman-proxy-certs.yaml
-%exclude %{_datadir}/%{name}/parser_cache/foreman-proxy-content.yaml
-%exclude %{_datadir}/%{name}/parser_cache/katello.yaml
-
-%files katello
-# common
+%dir %{_datadir}/%{name}
+%{_datadir}/%{name}/checks
+%dir %{_datadir}/%{name}/config
+%{_datadir}/%{name}/config/config_header.txt
+%{_datadir}/%{name}/config/foreman-hiera.yaml
+%{_datadir}/%{name}/config/foreman.hiera
+%{_datadir}/%{name}/hooks
+%{_datadir}/%{name}/modules
+%{_datadir}/%{name}/VERSION
+%dir %{parser_cache}
+%{_mandir}/man8/%{name}.8*
+
+%files katello-common
+%{_datadir}/%{name}/katello-certs
+%{parser_cache}/foreman-proxy-certs.yaml
```

Comment:
You may be right. I wasn't sure where we want to use this. Perhaps we shouldn't even name it an installer scenario and just make it `katello-certs` or something, since there's no separate scenario file it doesn't store any answers. It just provides a script to generate the certificates that could in theory also be used in some standalone server (like an offline CA).

Reply:
If we consider the downstream scenario then its own package makes sense.